dissect/target/loaders/tar.py

```diff
@@ -47,7 +48,7 @@ def map(self, target: target.Target):
                     target.filesystems.add(vol)
 
                 volume = volumes["/"]
-                entry = TarFile(volume, member.name, member.name, self.tar)
+                entry = TarFile(volume, member.name, member, self.tar)
```

Comment:
I believe the rest of the changes in this file have been superseded by https://github.com/fox-it/dissect.target/pull/112?

Reply:
Looks like it! Merged those changes into this branch